Gemfile.lock

```diff
@@ -302,3 +319,6 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   xray-rails
+
+BUNDLED WITH
```

Comment:
The last three lines here need to be cut out as they cause problems with Jenkins (everything after 1zray-rails`)